### PR TITLE
#161 update Setup and Configuration

### PR DESCRIPTION
**Pull request by devanshimodha**
Added useful links for the setup:

- Xcode documentation
- Xcode Source Control Management
- Combining code changes in a source control repository

### CODE REVIEW

File: coding-projects/ios/README.md [hunk 1]
```diff
@@ -6,8 +6,10 @@ This is an open source project to build a Timed Activity Tracker using SwiftUI.
 1. **Fork**: Click the "Fork" button to start your project based on what is currently available.
 
 2. **Setup and Configuration**:
-TODO:
-   - [Add specific setup steps here, e.g., installation instructions, environment setup, configuration steps, etc.]
+
+    [Xcode documentation](https://developer.apple.com/documentation/xcode)
+    [Xcode Source Control Management]([https:/](https://developer.apple.com/documentation/xcode/source-control-management)/)
```
Comment (ceceliacreates) [inline coding-projects/ios/README.md:11]:
```suggestion
    [Xcode Source Control Management](https://developer.apple.com/documentation/xcode/source-control-management)/)
```

**Review by ceceliacreates — CHANGES_REQUESTED** [review]
Just a small change, it looks like the formatting for the markdown link had some extra characters.

File: coding-projects/ios/README.md (hunk 1 above)
Comment (devanshimodha) [reply coding-projects/ios/README.md]: Fixed spacing @ceceliacreates

**Review by ceceliacreates — APPROVED** [review]

**Comment by ceceliacreates** [comment]
I am going to go ahead and merge, as this is a documentation change. Looks good @devanshimodha thanks for the fix

**Pull request merged**